From c4eb0ff86ece61afd8fb1deee8ccf44e7af54c01 Mon Sep 17 00:00:00 2001
From: Arjan van de Ven <arjan@linux.intel.com>
Date: Fri, 14 Apr 2017 13:35:53 +0000
Subject: [PATCH 114/114] cirrus: set ioremap_cache

make sure the fake framebuffer is not uncached

The cirrus driver is for KVM graphics..... no need for uncached accesses

in a virtual environment, the framebuffer is going to be dram backed anyway,
might as well use those cpu caches...
---
 drivers/gpu/drm/cirrus/cirrus_main.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/cirrus/cirrus_main.c b/drivers/gpu/drm/cirrus/cirrus_main.c
index b5f528543956..04e2574de8de 100644
--- a/drivers/gpu/drm/cirrus/cirrus_main.c
+++ b/drivers/gpu/drm/cirrus/cirrus_main.c
@@ -138,7 +138,7 @@ int cirrus_device_init(struct cirrus_device *cdev,
 		return -ENOMEM;
 	}
 
-	cdev->rmmio = ioremap(cdev->rmmio_base, cdev->rmmio_size);
+	cdev->rmmio = ioremap_cache(cdev->rmmio_base, cdev->rmmio_size);
 
 	if (cdev->rmmio == NULL)
 		return -ENOMEM;
-- 
2.16.1

